CMakeLists.txt

```diff
@@ -450,8 +450,15 @@ if (NOT CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
 endif()
 
 # Platform setup
-set(PLAYER_TARGET_PLATFORM "SDL2" CACHE STRING "Platform to compile for. Options: SDL2 libretro")
-set_property(CACHE PLAYER_TARGET_PLATFORM PROPERTY STRINGS SDL2 libretro)
+if(VITA)
```

Comment:
When building on psvita for libretro the toolchain file defines VITA making it impossible to compile for it.

Maybe remove this if and replace 

```set(PLAYER_TARGET_PLATFORM "SDL2" CACHE STRING "Platform to compile for. Options: SDL2 libretro")```

with

```set(PLAYER_TARGET_PLATFORM "SDL2" CACHE STRING "Platform to compile for. Options: SDL2 libretro psvita")```

and when here "psvita" is set do the

```
include("$ENV{VITASDK}/share/vita.cmake" REQUIRED)
set(CMAKE_DL_LIBS "") # hack4icu!
set(PLAYER_ENABLE_TESTS OFF)
```

-----------

Or require the user to just do ```-DCMAKE_TOOLCHAIN_FILE=VITASDK/share/vita.cmake``` and when ``VITA`` was set by this file expose "psvita" and "libretro" as options?

Reply:
Either way is fine. libretro was just not on my mind, but makes sense since they use the toolchain as well.

This likely applies for the big N toolchains as well, for them I have a similar pull request soon.